android/sdk/src/main/java/org/jitsi/meet/sdk/VideoConfBridge.java

```diff
@@ -4,8 +4,6 @@
 
 package org.jitsi.meet.sdk;
 
-import org.jitsi.meet.sdk.ReactInstanceManagerHolder;
```

Comment:
Doesn't this break the build process? To me this doesn't look like an unused import.